pkg/agent/openflow/pipeline.go

```diff
@@ -92,6 +92,27 @@ const (
 	ipv6LinkLocalAddr = "FE80::/10"
 )
 
+type ofAction int32
+
+const (
+	add ofAction = iota
+	mod
+	del
+)
+
+func (a ofAction) String() string {
+	switch a {
+	case 0:
```

Comment:
can it use `add`, `mod`, `del` to be more readable?

Reply:
Thanks for reminding. Updated.